openapi/components/schemas/QuoteCreateOrder.yaml

```
@@ -0,0 +1,468 @@
+type: object
+title: QuoteCreateOrder
+required:
+  - websiteId
+  - customerId
+  - items
+  - action
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote.
+      On accept quote new order is created. `orderId` and `invoiceId` are filled with new generated values from the order.
+    type: string
+    enum:
+      - create
+    example: create
+  acceptanceConditions:
+    type: array
+    writeOnly: true
+    description: If conditions are fulfilled the quote is automatically accepted.
```

```suggestion
    description: If conditions are fulfilled the quote is automatically accepted.
```

suggestion: for readability 
```suggestion
    description: Conditions that must be met for the quote to be accepted.
```